base/src/com/google/idea/blaze/base/sync/projectstructure/ModuleEditorImpl.java

```diff
@@ -141,29 +143,38 @@ private File getImlDirectory(BlazeImportSettings importSettings) {
     return new File(BlazeDataStorage.getProjectDataDir(importSettings), "modules");
   }
 
+  private static void removeImlFile(final VirtualFile imlFile) {
+    if (imlFile.isDirectory()) {
+      return;
+    }
+
+    ApplicationManager.getApplication()
+      .runWriteAction(
+          new Runnable() {
+            @Override
+            public void run() {
+              try {
+                imlFile.delete(this);
+              } catch (IOException e) {
+                logger.warn(
+                    String.format(
+                        "Could not delete file: %s, will try to continue anyway.",
+                        imlFile.getPath()),
+                    e);
+              }
+            }
+          });
+  }
+
+
   // Delete using the virtual file to ensure that IntelliJ properly updates its index.
   // Otherwise, it is possible for IntelliJ to read the
   // old IML file from its index and behave unpredictably
   // (like failing to save the new IML files to disk).
   private static void removeImlFile(final File imlFile) {
```

Comment:
Does this function get called anymore now? If not, could just modify the signature of it instead of overloading it?

Reply:
https://github.com/bazelbuild/intellij/pull/7726/files#diff-d8756c03d8c068257ad060a6be3b25b64c73c98bf95064cbcda35a623815ede4R80